build_oc.tool

```diff
@@ -268,16 +268,23 @@ package() {
   # Copy OpenDuetPkg booter.
   for arch in "${ARCHS[@]}"; do
     local tgt
-    local booter
+    local booter_bio
```

Comment:
Needs `local booter` here as well, since both vars are used!

On very minor point, would `booter_blockio` be a nicer name, just for consistency as that's the extension used everywhere else?